From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Tue, 11 Oct 2011 09:40:01 -0600
Patch-mainline: v3.2-rc1
Subject: [PATCH] hv: remove free_channel() from hyperv.h
Git-commit: 9f3e28e375a8d509a27efe89f3c8ea2a15aeb524

This function is only used in the file it is declared in
(channel_mgmt.c) so make it static and remove it from the hyperv.h file.

Cc: K. Y. Srinivasan <kys@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/hv/channel_mgmt.c |    2 +-
 include/linux/hyperv.h    |    2 --
 2 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/drivers/hv/channel_mgmt.c b/drivers/hv/channel_mgmt.c
index a2fc487..12b85ff 100644
--- a/drivers/hv/channel_mgmt.c
+++ b/drivers/hv/channel_mgmt.c
@@ -196,7 +196,7 @@ static void release_channel(struct work_struct *work)
 /*
  * free_channel - Release the resources used by the vmbus channel object
  */
-void free_channel(struct vmbus_channel *channel)
+static void free_channel(struct vmbus_channel *channel)
 {
 
 	/*
diff --git a/include/linux/hyperv.h b/include/linux/hyperv.h
index ae865a1..240e114 100644
--- a/include/linux/hyperv.h
+++ b/include/linux/hyperv.h
@@ -606,8 +606,6 @@ struct vmbus_channel {
 	void *channel_callback_context;
 };
 
-void free_channel(struct vmbus_channel *channel);
-
 void vmbus_onmessage(void *context);
 
 int vmbus_request_offers(void);
-- 
1.6.0.2


